qiskit/circuit/instruction.py

```diff
@@ -128,7 +128,11 @@ def __eq__(self, other):
 
     def _define(self):
         """Populates self.definition with a decomposition of this gate."""
-        pass
+        try:
+            self._definition = self.decompositions[0]
+        except IndexError:
+            # u3 or cx
+            pass
```

Comment:
is this change in the base class temporary?

Reply:
This allows not having to have a `_define` method in many standard_gates, so that the standard equivalence library is the only source of that information. If we eventually don't use `definition` at all then maybe it can go away.